### PR TITLE
test: refactor unit test acl manager

### PR DESCRIPTION
**Pull request by RealLau**
## Security Checklist

- [ ] 1. Re-Entrancy
- [ ] 2. Arithmetic Over/Under Flows
- [ ] 3. Unexpected Ether
- [ ] 4. Delegatecall
- [ ] 5. Default Visibilities
- [ ] 6. Entropy Illusion
- [ ] 7. External Contract Referencing
- [ ] 8. Short Address/Parameter Attack (off chain)
- [ ] 9. Unchecked CALL Return Values
- [ ] 10. Race Conditions / Front Running
- [ ] 11. Denial Of Service (DOS)
- [ ] 12. Block Timestamp Manipulation
- [ ] 13. Constructors with Care
- [ ] 14. Uninitialized Storage Pointers
- [ ] 15. Floating Points and Precision
- [ ] 16. Tx.Origin Authentication
- [ ] 17. Address.isContract Re-Entrancy via Constructor

### ⚠️ NOTES ⚠️

Make sure to think about each of these exploits in this PR.

### CODE REVIEW

File: test-suites/_base_acl_manager.spec.ts [hunk 1]
```diff
@@ -1,127 +1,372 @@
...
-    utils.formatBytes32String("FLASH_BORROWER_ADMIN")
+const fixture = async () => {
+  const testEnv = await loadFixture(testEnvFixture);
+  const {deployer, addressesProvider} = testEnv;
+  const aclManager = await new ACLManager__factory(deployer.signer).deploy(
+    addressesProvider.address
   );
+return {
+  aclManager,
+  ...testEnv
```
Comment (alannotnerd) [inline test-suites/_base_acl_manager.spec.ts:18]:
```suggestion
 ...testEnv,
  aclManager
```

**Review by alannotnerd — CHANGES_REQUESTED** [review]
Pls resolve the CI issue and the conflicts.

**Review by alannotnerd — CHANGES_REQUESTED** [review]
Pls resolve the CI failure

**Comment by ivansolomonoff** [comment]
Make sure to remove the original test script file within this PR as well.

**Pull request closed without merging**